arm-eventhub/2015-08-01/swagger/EventHub.json

```diff
@@ -1074,35 +1192,43 @@
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The name of the resource group."
+            "minLength": 1,
+            "maxLength": 90,
+            "description": "The name of the Resource Group."
           },
           {
             "name": "namespaceName",
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The namespace name."
+            "minLength": 6,
+            "maxLength": 50,
+            "description": "The Namespace Name within the Resource Group"
```

Comment:
The name of the Namespace (with in the resource group can be removed, as even outside it is same)